api/HTMLElement.json

```diff
@@ -388,7 +388,8 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-beforetoggle",
           "support": {
             "chrome": {
-              "version_added": "114"
+              "version_added": "114",
+              "notes": "Only available on elements that include the <code>popover</code> attribute."
```

Comment:
Is this behavior per spec (then why do we have this note?) or is it a bug in Chrome? (should we add partial_implementation then?)